dashboard/src/features/database/settings/components/DatabaseMigrateLogsModalText/DatabaseMigrateLogsModalText.tsx

```diff
@@ -0,0 +1,71 @@
+import { Text } from '@/components/ui/v2/Text';
+import { useMigrationLogs } from '@/features/database/common/hooks/useMigrationLogs';
+
+export default function DatabaseMigrateLogsModalText() {
```

Comment:
nitpick: I can see this was only referenced once so no real need to put it in it's own separate component, can we move this back to the main `DatabaseMigrateLogsModal`